### PR TITLE
Update aria labels and amends on selectors for e2e

### PR DESCRIPTION
**Pull request by ishouty**
* update test selectors
* fix up aria labels
* add arial labels for search form
* update actions in e2e tests

### CODE REVIEW

File: catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx [hunk 1]
```diff
@@ -308,9 +308,9 @@ const ExpandedImage: FunctionComponent<Props> = ({
         });
 
   return (
-    <div role="dialog" aria-labelledby={ariaLabelledBy} aria-modal={true}>
+    <div role="dialog" id={ariaLabelledBy} aria-modal={true}>
       <Overlay onClick={() => setExpandedImage(undefined)} />
-      <Modal ref={modalRef}>
+      <Modal ref={modalRef} aria-labelledBy={ariaLabelledBy}>
```
Comment (gestchild) [inline catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx:313]: Don't think we're using aria-labelledBy correctly here. The value should be the id of an element(s) that provide the text needed for labeling. I don't think the dialog div does that. See [using aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute)

Comment (gestchild) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx:313]: Also, aria-labelledby should all be lowercase

Comment (ishouty) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx:313]: gotcha, i think i will remove this. thanks for this.

File: catalogue/webapp/test/e2e/actions/search.ts [hunk 2]
```diff
@@ -58,19 +60,28 @@ export async function clickActionFormatRadioCheckbox(
...
   await page.click(selector);
 }
 
 export async function clickActionCloseModalFilterButton(
   condition?: conditionSearchType
 ): Promise<void> {
-  const selector = `${
-    condition === 'images' ? searchImagesForm : searchWorksForm
-  } ${mobileModalCloseButton}`;
+  const searchImagesCloseButton = `${searchImagesForm} ${mobileModalCloseButton}`;
```
Comment (jamesgorrie) [inline catalogue/webapp/test/e2e/actions/search.ts:77]: Should this just be stored in the selectors file?

File: catalogue/webapp/test/e2e/selectors/search.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
 // input
 
-export const worksSearchInputField = 'input[aria-label="search input field"]';
+export const worksSearchCatalogueInputField =
+  'input[aria-label="Search the catalogue"]';
```
Comment (jamesgorrie) [inline catalogue/webapp/test/e2e/selectors/search.ts:4]: You could pull this out of the labels file.

**Review by jamesgorrie — APPROVED** [review]

File: catalogue/webapp/test/e2e/actions/search.ts (hunk 2 above)
Comment (ishouty) [reply catalogue/webapp/test/e2e/actions/search.ts:77]: very good point .. moving it over

Comment (ishouty) [reply catalogue/webapp/test/e2e/actions/search.ts:77]: when moving over these variables , i had issues with the selectors being undefined. Ill leave this for now and can work on this after.

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**